app/shared/models/git_repo.rb

```diff
@@ -466,17 +545,22 @@ def clone_synchronously(repo_auth: self.repo_auth)
       # git_config.id.length > 1 to ensure we're not empty or a space
       if git_config.id.length > 1 && Dir.exist?(existing_repo_for_project)
         logger.debug("Removing existing repo at: #{existing_repo_for_project}")
-        require "fileutils"
+
         # Danger zone
         FileUtils.rm_r(existing_repo_for_project)
       end
 
       self.temporary_storage_path = setup_auth(repo_auth: repo_auth)
       logger.debug("[#{git_config.id}]: Cloning git repo #{git_config.git_url} to #{@local_folder}")
-      Git.clone(git_config.git_url,
-                "", # checkout into the local_folder
-                path: local_folder,
-                recursive: true)
+
+      begin
+        Git.clone(git_config.git_url,
```

Comment:
This is a nit, but could we potentially style it as this instead:

```ruby
Git.clone(
  git_config.git_url,
  "", # checkout into the local_folder
  path: local_folder,
  recursive: true
)  
```

Thoughts?

Reply:
I agree 🌮